asset-transfer-private-data/chaincode-go/META-INF/statedb/couchdb/collections/collectionMarbles/indexes/indexOwner.json

```diff
@@ -0,0 +1 @@
+{"index":{"fields":["docType","owner"]},"ddoc":"indexOwnerDoc", "name":"indexOwner","type":"json"}
```

Comment:
File path still includes 'collectionMarbles'. Use the new collection name.
Are you planning to add a JSON query?

Reply:
Calling it ownerCollection

Reply:
assetCollection